core-rust-bridge/src/main/java/com/radixdlt/statecomputer/commit/CommitRequest.java

```diff
@@ -92,32 +88,27 @@ public boolean equals(Object o) {
     if (o == null || getClass() != o.getClass()) return false;
     CommitRequest that = (CommitRequest) o;
     return Objects.equals(transactions, that.transactions)
-        && Objects.equals(stateVersion, that.stateVersion)
-        && Objects.equals(stateHash, that.stateHash)
-        && Arrays.equals(proofBytes, that.proofBytes)
-        && Objects.equals(postCommitVertexStoreBytes, that.postCommitVertexStoreBytes);
```

Comment:
_(note: this change is unrelated, but I believe it has always been a bug [equality on `byte[]` identity])_

Reply:
huh, nice catch